### ISSUE TITLE
UHD620 better ig-platform-id

### ISSUE THREAD

**Issue by rossna** [the-Quert/XPS-9360-macOS#49]
For those of us with the i7-8550u version, this ig-platform-id gives us better graphics performance:

ig-platform-id: 0900A53E
platform-id: A53E0000

[Source](https://dortania.github.io/GPU-Buyers-Guide/modern-gpus/intel-gpu.html#kabylake-refresh-coffee-lake-cometlake-8xxx-9xxx)

The old platform-id (00001659) is basically patching our iGPU to act as an HD520, which it is not.

**Comment by egore24** [comment]
Good tip. Trying it now. Thanks!

Just a note, 
platform-id: A53E0000   should be  device-id: A53E0000

**Comment by rolsanmordua** [comment]
@rossna brightness controls seem to break for me with this change

**Comment by rolsanmordua** [comment]
Assuming no other change was necessary, I ran the Geekbench 5 compute test and my Metal score dropped ≈250 points from before

**Comment by jkbuha** [comment]
Yeah had the same issue too. It's because the existing BRT6 SSDT is mapped to device-id 1659, so the PNLF fails to catch that. The SSDT overrides for the GPU will need to be patched as well.

Regardless though, there is a performance drop with this new id.

Best config layout I've optimised so far can be found on my post here: https://www.tonymacx86.com/threads/guide-dell-xps-13-9360-on-macos-sierra-10-12-x-lts-long-term-support-guide.213141/page-332

If anyone has any better performance suggestions please reach out.

**Comment by mankot14** [comment]
> Yeah had the same issue too. It's because the existing BRT6 SSDT is mapped to device-id 1659, so the PNLF fails to catch that. The SSDT overrides for the GPU will need to be patched as well.
> 
> Regardless though, there is a performance drop with this new id.
> 
> Best config layout I've optimised so far can be found on my post here: https://www.tonymacx86.com/threads/guide-dell-xps-13-9360-on-macos-sierra-10-12-x-lts-long-term-support-guide.213141/page-332
> 
> If anyone has any better performance suggestions please reach out.

Hey, I saw your post on tonymac. I have the i7-8550u with UHD 620 variant just as you do. The best setting for graphics I've found are as follows:
AAPL,ig-platform-id = 00001B59 
device-id = 16590000 
I got a score of just over 4900 on battery power for OpenCL in Geekbench5 with this.

**Comment by linianqing** [comment]
> > Yeah had the same issue too. It's because the existing BRT6 SSDT is mapped to device-id 1659, so the PNLF fails to catch that. The SSDT overrides for the GPU will need to be patched as well.
> > Regardless though, there is a performance drop with this new id.
> > Best config layout I've optimised so far can be found on my post here: https://www.tonymacx86.com/threads/guide-dell-xps-13-9360-on-macos-sierra-10-12-x-lts-long-term-support-guide.213141/page-332
> > If anyone has any better performance suggestions please reach out.
> 
> Hey, I saw your post on tonymac. I have the i7-8550u with UHD 620 variant just as you do. The best setting for graphics I've found are as follows:
> AAPL,ig-platform-id = 00001B59
> device-id = 16590000
> I got a score of just over 4900 on battery power for OpenCL in Geekbench5 with this.

Is your brightness controls ok?

**Comment by mankot14** [comment]
> > > Yeah had the same issue too. It's because the existing BRT6 SSDT is mapped to device-id 1659, so the PNLF fails to catch that. The SSDT overrides for the GPU will need to be patched as well.
> > > Regardless though, there is a performance drop with this new id.
> > > Best config layout I've optimised so far can be found on my post here: https://www.tonymacx86.com/threads/guide-dell-xps-13-9360-on-macos-sierra-10-12-x-lts-long-term-support-guide.213141/page-332
> > > If anyone has any better performance suggestions please reach out.
> > 
> > 
> > Hey, I saw your post on tonymac. I have the i7-8550u with UHD 620 variant just as you do. The best setting for graphics I've found are as follows:
> > AAPL,ig-platform-id = 00001B59
> > device-id = 16590000
> > I got a score of just over 4900 on battery power for OpenCL in Geekbench5 with this.
> 
> Is your brightness controls ok?

I went back to AAPL,ig-platform-id = 00001659 and device-id = 16590000 because that's the only configuration that allowed me to have external display AND full functionality of the laptop display.

**Comment by kaedea** [comment]
> > Yeah had the same issue too. It's because the existing BRT6 SSDT is mapped to device-id 1659, so the PNLF fails to catch that. The SSDT overrides for the GPU will need to be patched as well.
> > 
> > Regardless though, there is a performance drop with this new id.
> > 
> > Best config layout I've optimised so far can be found on my post here: https://www.tonymacx86.com/threads/guide-dell-xps-13-9360-on-macos-sierra-10-12-x-lts-long-term-support-guide.213141/page-332
> > 
> > If anyone has any better performance suggestions please reach out.
> 
> Hey, I saw your post on tonymac. I have the i7-8550u with UHD 620 variant just as you do. The best setting for graphics I've found are as follows:
> AAPL,ig-platform-id = 00001B59 
> device-id = 16590000 
> I got a score of just over 4900 on battery power for OpenCL in Geekbench5 with this.

Did your external HDMI display work with 1B59?

**Comment by kaedea** [comment]
> For those of us with the i7-8550u version, this ig-platform-id gives us better graphics performance:
> 
> ig-platform-id: 0900A53E
> platform-id: A53E0000
> 
> [Source](https://dortania.github.io/GPU-Buyers-Guide/modern-gpus/intel-gpu.html#kabylake-refresh-coffee-lake-cometlake-8xxx-9xxx)
> 
> The old platform-id (00001659) is basically patching our iGPU to act as an HD520, which it is not.

Can not boot with this config.
